index.html

```diff
@@ -0,0 +1,968 @@
+        </div>
+        <div class="left-navbar">
+          <a href="#">LATEST</a>
+          <a href="#">HARD FORK</a>
+          <a href="#">PLUGGED</a>
+          <a href="#">README</a>
+          <a href="#">GROWTH QUARTERS</a>
+          <a href="#">SHIFT</a>
+          <a href="#">NEURAL</a>
+        </div>
+        <div class="right-nav flex">
+          <i class="fas fa-search"></i>
+          <img src="./images/logo1.png" alt="logo" />
+          <img src="./images/clock.PNG" alt="clock" />
+          <button type="submit">GET TICKETS</button>
+        </div>
+      </div>
+      <div class="announcement">
+        <a href="#">
+          Join TNW2020, our 15th annual flagship event. Online. Get tickets ➔
+        </a>
+      </div>
+    </header>
+    <main>
+      <!-- section 1 -->
+      <section class="section1">
+        <div class="bigimg sectionflex">
+          <div class="overlay"></div>
+          <div class="sectionflex bigimgtop">
+            <div id="TNW">
+              <h1>TNW</h1>
+              <p>Tuesday-August 25, 2020</p>
+            </div>
+          </div>
+
+          <div class="link">
+            <p>
+              <a href="#"
+                >Two left feet? Facebook’s new AI dance choreographer can teach
+                you some brand new moves</a
+              >
+            </p>
+            <div class="sectionflex">
+              <p>by Thomas Macaulay</p>
+              <i class="far fa-clock"></i>
+              <p>1 hour ago</p>
+              <i class="fas fa-share-alt"></i>
+              <p>8</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="smallimg">
+          <div class="righttop sectionflex">
+            <p><a href="#">Join our 15th annual flagship event for free</a></p>
+          </div>
+
+          <div class="rightbottom sectionflex">
+            <p>
+              <a href="#"
+                >Nintendo reportedly planning a beefed-up Switch console for
+                2021</a
+              >
+            </p>
+            <div class="sectionflex white">
+              <p>by Mix</p>
+              <i class="far fa-clock"></i>
+              <p>2 hours ago</p>
+              <i class="fas fa-share-alt"></i>
+              <p>3</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- section2 -->
+      <section class="alignment">
+        <div class="title">
+          <h1>Latest news</h1>
+          <i class="fas fa-angle-right"></i>
+        </div>
+        <div class="grid grid-divs">
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
```

Comment:
#### Mobile Screen Size

- [ ] The author's name here is with the wrong styling.

- [ ] The image can be smaller

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91436463-8da15b80-e83e-11ea-9a39-8857b759d0f1.png)| ![image](https://user-images.githubusercontent.com/61479648/91436487-9b56e100-e83e-11ea-9574-ae4dbee1a3ad.png)  |